cmd/cosign/cli/env.go

```diff
@@ -21,29 +21,41 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/pkg/cosign/env"
 )
 
 func Env() *cobra.Command {
-	return &cobra.Command{
+	o := &options.EnvOptions{}
+
+	cmd := &cobra.Command{
 		Use:   "env",
 		Short: "Prints Cosign environment variables",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			for _, e := range getEnv() {
+			// Print COSIGN_ environment variables
+			env.PrintEnv(o.ShowDescriptions, o.ShowSensitiveValues)
+
+			// Print external environment variables (SIGSTORE_ and TUF_)
+			for _, e := range getExternalEnv() {
 				fmt.Println(e)
```

Comment:
This is just printing the name, right? Want to avoid sensitive values I think.

Reply:
No, this is printing `key=value`. We can't really tell if it's sensitive or not in this case.